CHANGELOG.md

```diff
@@ -14,11 +14,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Added
```

Comment:
Are we missing the nerd program from here?

Reply:
I have added the description and template README for nerd https://github.com/SMI/SmiServices/commit/0691cbab955cf5652fe02b70d85e55c3ca60929e @jas88 can you fill in the readme to show how to configure/run it?

Thanks.